includes/class-wc-stripe-webhook-handler.php

```
@@ -1391,6 +1391,86 @@ public function process_account_updated( $notification ) {
+		if ( $order_helper->lock_order_payment( $order ) ) {
+			return;
+		}
+
+		$intent_id = $checkout_session->payment_intent;
+
+		// Store the payment intent ID on the order.
+		$order_helper->add_payment_intent_to_order( $intent_id, $order );
+
+		// Add presentment details if available.
+		if ( ! empty( $checkout_session->presentment_details ) ) {
+			$presentment_details = $checkout_session->presentment_details;
```

PHPMD (2.15.0)

[error] The variable $presentment_details is not named in camelCase. (undefined)

(CamelCaseVariableName)